### ISSUE TITLE
Simulator operations and use by Hackbots

### ISSUE THREAD

**Issue by bczdog** [hackbots-3414/2021-Infinite_Recharge_At_Home#19]
Get the 2021 code operational in the WPILib Simulator, including the TalonFX / TalonSRX motors and SparkMax if possible.  Need to understand what sensors like the ultrasonic and gyro / NavX look like and how we can test them in simulation.

**Comment by VSingh1012** [comment]
Would we need to import existing code and make changes and check for errors to be able to simulate it?

**Comment by bczdog** [comment]
Yep.  Clone this repository, do a build (make sure it builds successfully), create a branch (git checkout -b _____your_branch_name___).  Once you have that, follow: https://docs.wpilib.org/en/stable/docs/software/wpilib-tools/robot-simulation/index.html.
